features/admin_center_manage_display.feature

```diff
@@ -652,14 +652,51 @@ Feature: Manage Display
     Then I should be able to access 3 "new_secondary_header_link" rows
     And I should not see "Add new secondary header link"
 
-  Scenario: Editing the Visual Design Settings when "Show Redesign Display Settings" and "Use Redesigned Results Page" are true
+  Scenario: Display sub navigation links when "Show Redesign Display Settings" and "Use Redesigned Results Page" are true/false
     Given the following Affiliates exist:
       | display_name | name       | contact_email   | first_name | last_name | show_redesign_display_settings | use_redesigned_results_page |
       | agency site  | agency.gov | john@agency.gov | John       | Bar       | true                           | true                        |
     And I am logged in with email "john@agency.gov"
+    And I go to the agency.gov's Manage Display page
```

Comment:
Is this needed?

Reply:
For other scenarios it is needed as user need to go to 'Display page' to see other sub navigation links. But for this scenario Visual Design page is enough to view the sub navigation links.